.github/workflows/remove-automerge-label.yml

```diff
@@ -0,0 +1,33 @@
+name: Remove Auto-Merge Label on New Commit
+
+on:
+  pull_request:
+    types:
+      - synchronize
+
+jobs:
+  remove-label:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    if: ${{ contains(fromJSON('["OWNER", "MEMBER", "COLLABORATOR"]'), github.event.pull_request.author_association )}}
```

We can skip this job if the PR author is not a contributor.

Reply:
Do we also want to skip for owners?

Reply:
I'm an owner.

Reply:
Ok, then I think whitelisting is better. Could you take a look at my change?

Reply:
Can we trust `CONTRIBUTOR`?

Reply:
> in case github introduces new associations?

If github does this, we can update our code.

Reply:
>Can we trust CONTRIBUTOR? Can we trust a person who fixed a typo?

Sorry it was a mistake, I've updated to COLLABORATOR

Reply:
> actually I think we're saying the same thing, your example just missed "!" in front of the contain.

Ah, sorry...